docs/source/platform/operation-registry.md

```diff
@@ -290,6 +290,50 @@ const server = new ApolloServer({
 });
 ```
 
+## Migrating from 0.1-alpha.4
+
+#### On servers that safelist operations
+
+- Upgrade `apollo-server` to `2.6.2` and `apollo-server-plugin-operation-registry` to `0.2.0-alpha.1` (there are no breaking changes)
+
+#### To target graph variants other than "current" (default, no tag)
```

Comment:
Does it make sense to connect "current" and (default, no tag) more? Maybe with no space between?

Totally could be nitpicky, since it's a small optimization. Also might not be possible to misconstrue what in the title is the default/no tag, so feel free to ignore